presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -84,6 +84,7 @@
     public static final String FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_SIZE = "filter_and_project_min_output_page_size";
```

Comment:
Commit message: `Add property that allows to disable sort redistribution` did you mean `enable`?